examples/baselines/ppo/ppo_fast.py

```diff
@@ -86,6 +86,8 @@ class Args:
     """frequency to save training videos in terms of iterations"""
     control_mode: Optional[str] = "pd_joint_delta_pos"
     """the control mode to use for the environment"""
+    robot_uids: Optional[str] = None
```

Comment:
thanks for modifying and testing the PPO code! 

do you know if tyro cli supports generic kwargs to pass via cli? I guess there are a lot of env kwargs like control mode robot uids etc that one can pass but I would prefer not to have a line of code for each.

Reply:
np, ty for maintaining this repo, legit one of the best sim stacks for manip

dunno, there probably is some way to do what you are describing, it is definitely ugly/verbose to just keep appending kwargs. BUT for the sake of this PR I think its better to keep it minimal and separately create some kind of github issue/ticket thing for the cleaner kwargs